common/comms/common_https.py

```diff
@@ -61,4 +63,8 @@ class CommonHttps(object):
                         "proxy_port": http_proxy_port,
                         "code": response.code
                     })
+
+        if logger.isEnabledFor(logging.DEBUG):
```

Comment:
Need this check because `decode()` fails on mock and it's easier to add this check there than modify tons of tests

Reply:
Would it be better to leave out logging of the body? I think the underlying http client will do this if enabled. There is also the warning at startup about logging clinical data.

Reply:
The warning says:
```The current log level (NOTSET) is set below INFO level, it is known that libraries used by this application sometimes log out clinical patient data at DEBUG level. The log level provided MUST NOT be used in a production environment.```
I think it's ok to actually put this request+response in DEBUG

Reply:
DEBUG not used for anything else, no reason to enable in production. Other libraries could potentially log out similar data if DEBUG were enabled also.